src/resource_dasm.cc

```diff
@@ -75,6 +81,65 @@ class ResourceExporter {
       }
     }
   }
+  
+  string output_filename(
+      const string& base_filename,
+      const int16_t* res_id,
+      const string& res_type,
+      const string& res_name,
+      const std::string& after) {
+    if (base_filename.empty()) {
+      return out_dir;
+    }
+
+    string base_out_dir_str = this->base_out_dir;
+    if (!base_out_dir_str.empty()) {
+      base_out_dir_str += '/';
+    }
+    string filename_str;
+    if (!this->out_dir.empty()) {
+      filename_str += this->out_dir;
+      filename_str += '/';
+    }
+    filename_str += base_filename;
+    
+    string  result = base_out_dir_str;
+    bool    saw_percent = false;
+    for (char ch : this->filename_format) {
+      if (ch == '%' && !saw_percent) {
+        saw_percent = true;
+      } else if (saw_percent) {
+        saw_percent = false;
+        switch (ch) {

```

Comment:
The %% code is documented in the usage text but doesn't appear to be implemented here.

Reply:
I'm so sorry, I totally forgot to test '%%'. Fixed, of course.